lesson12/js/main.js

```diff
@@ -25,6 +25,7 @@ function removeInnerHTML(element) {
 function createMenuList(data) {
 	const fragment = document.createDocumentFragment();
 	removeInnerHTML(menuLists);
+	console.log(data);
 
 	data.forEach((value) => {
 		const listItem = document.createElement('li');
```

Comment:
MUST
I don't think this file is needed.
too `lesson15/js/main.js `

Reply:
Thank you for your review.
I deleted a file that I had committed by mistake.

[68d6274](https://github.com/nyattsu72/hanson-frontend-cource/pull/21/commits/68d6274798d338a26338bd74edd09690096834cd)